src/Parser/Druid/Restoration/Constants.js

```diff
@@ -11,14 +11,12 @@ export const ABILITIES_AFFECTED_BY_HEALING_INCREASES = [
   SPELLS.CENARION_WARD_HEAL.id,
   SPELLS.LIFEBLOOM_HOT_HEAL.id,
   SPELLS.LIFEBLOOM_BLOOM_HEAL.id,
-  SPELLS.DREAMWALKER.id,
   SPELLS.SWIFTMEND.id,
-  SPELLS.NATURES_ESSENCE_DRUID.id,
   SPELLS.RENEWAL_TALENT.id,
   SPELLS.SPRING_BLOSSOMS.id,
   // The following spells don't double dip in healing increases.
   SPELLS.LEECH.id,
-  SPELLS.DREAMER.id,
+  //TODO - blazyb
```

Comment:
It would be nice if you could specify what's to do so that maybe someone could help